src/core/send.c

```diff
@@ -1120,6 +1121,16 @@ QuicSendFlush(
         return TRUE;
     }
 
+    //
+    // Connection CID changes on idle state after an amount of time
+    //
+    if (Connection->Settings.DestCidUpdateIdleTimeoutMs != 0 &&
+        Send->LastFlushTimeValid &&
+        CxPlatTimeDiff64(Send->LastFlushTime, TimeNow) >= MS_TO_US(Connection->Settings.DestCidUpdateIdleTimeoutMs)) {
+        Connection->Stats.Misc.DestCidUpdateCount++;
+        QuicConnRetireCurrentDestCid(Connection, Path);
```

Comment:
```suggestion
        (void)QuicConnRetireCurrentDestCid(Connection, Path);
```
We use this pattern to explicitly show we are ignoring the return value.